forward_engineering/helpers/general.js

```diff
@@ -86,6 +86,31 @@ module.exports = app => {
 		return options.length ? `\n OPTIONS(\n${tab(options.join(',\n'))}\n)` : '';
 	};
 
+	const foreignKeysToString = keys => {
+		if (Array.isArray(keys)) {
+			const activatedKeys = keys
+				.filter(key => _.get(key, 'isActivated', true))
+				.map(key => key.name.trim());
+			const deactivatedKeys = keys
+				.filter(key => !_.get(key, 'isActivated', true))
+				.map(key => key.name.trim());
+			const deactivatedKeysAsString = deactivatedKeys.length
+				? commentIfDeactivated(deactivatedKeys, { isActivated: false }, true)
+				: '';
+
+			return activatedKeys.join(', ') + deactivatedKeysAsString;
+		}
+		return keys;
+	};
+
+	const foreignActiveKeysToString = keys => {
+		return keys.map(key => key.name.trim()).join(', ');
+	};
+
+	const getTableName = ({tableName, datasetName}) => {
```

Comment:
Are you sure that we don't have anything like that? 
```
const tableName = getFullName(dbData.projectId, dbData.databaseName, name);
```
If, for some reason you cannot use it, make sure that we encode the name with `